Makefile

```diff
@@ -61,7 +61,22 @@ METRICS_PORT ?= 9118
 # Preferred Capella version
 CAPELLA_VERSION ?= 5.2.0
 
-DOCKER_TAG ?= latest
+# Should be 'latest', the branch name, the commit hash or a Git tag name
+CAPELLA_DOCKERIMAGES_VERSION ?= latest
+
+# Capella build type (online/offline)
+CAPELLA_BUILD_TYPE ?= online
+
+DOCKER_BUILD_FLAGS ?=
+
+# If set to 1, we will push the images to the specified registry
+PUSH_IMAGES ?= 0
+
+# Registry to push images
+LOCAL_REGISTRY_NAME ?= localhost
+LOCAL_REGISTRY_PORT ?= 12345
```

Comment:
I'd like to understand why these vars are prefixed with `LOCAL_`?

Reply:
Good point! We should change it. I think it makes sense to have one variable `DOCKER_REGISTRY`. What do you think?

Reply:
Yes, I like that proposal. Just registry

Reply:
Done.